src/backend/web/routes/index.js

```diff
@@ -1,13 +1,10 @@
 const express = require('express');
 const path = require('path');
-const admin = require('./admin');
 
 const router = express.Router();
 
-router.get('/', (req, res) => {
-  res.sendFile(path.join(__dirname, '../../../frontend/index.html'));
-});
+router.use(express.static(path.join(__dirname, '../../../frontend')));
 
-router.use('/admin', admin);
+router.use('/admin', require('./admin'));
```

Comment:
We are keeping module imports at the top of the file. See [Node style best practices](https://github.com/goldbergyoni/nodebestpractices#-38-require-modules-first-not-inside-functions). Please change it back.